contracts/ITokenGeyser.sol

```diff
@@ -9,7 +9,7 @@ import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 interface ITokenGeyser {
     function stake(uint256 amount) external;
     function unstake(uint256 amount) external returns (uint256);
-    function totalStakedFor(address addr) external view returns (uint256);
+    function totalStakedBy(address addr) external view returns (uint256);
```

Comment:
I like the name change, but flagging that it would mean changes for the frontend. Assuming it would need to be smart enough to know which contract version it's talking to on the backend.

Reply:
Right. I think for the new frontend we shouldn't expect backward compatibility with OG v1